kafka/client_async.py

```diff
@@ -179,6 +181,7 @@ class KafkaClient(object):
         'sasl_plain_username': None,
         'sasl_plain_password': None,
         'sasl_kerberos_service_name': 'kafka',
+        'kerberos_domain_name': None
```

Comment:
Let's maintain the `sasl_` prefix here (so `sasl_kerberos_domain_name`)

Reply:
OK, thank you.